kernel/environ.mli

```diff
@@ -386,10 +387,10 @@ val push_context_set : ?strict:bool -> ContextSet.t -> env -> env
     context set to the environment. It does not fail even if one of the
     universes is already declared. *)
 
-val push_qualities : Sorts.QVar.Set.t -> env -> env
-(** [push_qualities qs env] pushes the set of quality variables in
-    the environment. It fails if a quality variable is already
-    declared. *)
+val push_quality_set : Sorts.QVar.Set.t -> env -> env
```

Comment:
rebase went wrong? this probably shouldn't have changed (note that the comment is wrong, the current version of the PR does fail is a qvar is already declared)